### ISSUE TITLE
discrepancy between record numbers in library list view and actual records in individual library

### ISSUE THREAD

**Issue by ehenneken** [adsabs/biblib-service#114]
The call
```
https://ui.adsabs.harvard.edu/v1/biblib/libraries
```
returns data used to create the library overview `All ADS Libraries`. The numbers in the column `PAPERS` do not always agree with the actual number of records in individual libraries. The numbers in the `PAPERS` column are the values of `num_documents` in the entries for the individual libraries listed in the value of the `libraries` attribute in the microservice response. When querying for the contents of an individual library, the microservice response has a `"solr"` attribute, which has a `"response"` attribute, containing the number of records in `"numFound"`.

The overview of libraries is created in the `UserView`, while contents of individual libraries are interacted with in the `LibraryView`. The actual number of records in a library only gets "discovered" when the bibcodes in a library are sent to Solr through a bigquery.

In short, in the current implementation, we cannot get an accurate record count for a library until we send the bibcodes in that library to Solr. It seems unadvisable to fire off a bigquery to Solr just to get accurate counts on the `All ADS Libraries` view. We also have to be careful with doing library maintenance in the background with regards to updating them (like record merges, deletions).

Possible solutions are to updated the list of stored bibcodes to reflect changes (dangerous!), or to calculate `num_documents` not as the length of the list of bibcodes, but to "canonicalize" that list first. This number could be updated nightly via a `cron` job, leaving the list of bibcodes untouched.
